data-model/fdo-types/0.3.0/digital-media-objects/schema/digital-entity.json

```diff
@@ -0,0 +1,195 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/fdo-types/0.3.0/digital-media-objects/digital-entity.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$comment": "DigitalObject Version 0.3.0",
+  "type": "object",
+  "properties": {
+    "ods:id": {
+      "type": "string",
+      "description": "The unique digital identifier of the object",
+      "pattern": "https:\/\/hdl.handle.net\/20.5000.1025\/(.){3}-(.){3}-(.){3}",
+      "examples": [
+        "https://hdl.handle.net.org/20.5000.1025/XXX-XXX-XXX"
+      ],
+      "$comment": "Does an image get a DOI or a handle?"
+    },
+    "ods:version": {
+      "type": "integer",
+      "description": "The version of the object, each change generates a new version",
+      "minimum": 0,
+      "examples": [
+        1
+      ]
+    },
+    "ods:created": {
+      "type": "string",
+      "description": "The timestamp that the object version was created in DiSSCo",
+      "format": "date-time",
+      "examples": [
+      ]
+    },
+    "ods:type": {
+      "type": "string",
+      "description": "The FDO type of the object",
+      "$comment": "Unclear what value goes here"
+    },
+    "dcterms:type": {
+      "type": "string",
+      "description": "https://purl.org/dc/terms/type",
+      "enum": [
+        "Collection",
+        "Dataset",
+        "Event",
+        "Image",
+        "InteractiveResource",
+        "MovingImage",
+        "PhysicalObject",
+        "Service",
+        "Software",
+        "Sound",
+        "StillImage",
+        "Text"
+      ]
+    },
+    "ac:accessUri": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/ac/terms/accessURI",
+      "examples": [
+      ]
+    },
+    "dwc:institutionId": {
+      "type": "string",
+      "description": "ROR or Wikidata identifier, based on https://rs.tdwg.org/dwc/terms/institutionID",
+      "examples": [
+        "https://ror.org/015hz7p22"
+      ],
+      "$comment": "Add format for ROR or Wikidata ID"
```

Comment:
`(https:\/\/ror\.org\/0[\w]{6}[0-9]{2})|(http:\/\/www\.wikidata.org\/entity\/\w+)`

Reply:
Thanks forgot about this one

Reply:
Validator is not very happy with this regex

Reply:
Need a double escape `\` https://stackoverflow.com/questions/30739738/json-schema-validation-with-escaped-characters-in-patterns-fails

Reply:
oops was missing an escape in wikidata.org, this works:

`(https:\/\/ror\.org\/0[\w]{6}[0-9]{2})|(http:\/\/www\.wikidata\.org\/entity\/\w+)
`
https://regex101.com/r/PwZs3C/1